src/XamlStyler.dotUltimate/src/dotnet/ReSharperPlugin.XamlStyler.dotUltimate/Options/StringListViewModel.cs

```diff
@@ -0,0 +1,79 @@
+using System;
+using System.Linq;
+using JetBrains.Annotations;
+using JetBrains.DataFlow;
+using JetBrains.Lifetimes;
+using JetBrains.Threading;
+using JetBrains.Util;
+
+namespace ReSharperPlugin.XamlStyler.dotUltimate.Options
+{
+    public class StringListViewModel
```

Comment:
A Rider/R# compatible user control model to list strings in a table. Check the options page to see where it is used.